test-plone-5.1.x.cfg

```diff
@@ -0,0 +1,17 @@
+[buildout]
+extends =
+    https://raw.github.com/4teamwork/ftw-buildouts/master/test-plone-5.1.x.cfg
+    sources.cfg
+
+package-name = ftw.mobile
+
+extensions +=
+    mr.developer
+
+auto-checkout =
+    ftw.theming
+    ftw.gopip
```

Comment:
Why isn't that in `sources.cfg`?

Reply:
I have move `ftw.theming` checkout to `sources.cfg`. I have removed the `ftw.gopip` checkout because a Plone 5.1 compatible version of `ftw.gopip` has been released.